### DISCUSSION TITLE
Show the time plotting started and estimated finish time

### DISCUSSION THREAD

**Discussion by Isanderthul** [Chia-Network/chia-blockchain#4427] [Ideas]
If would be nice to have an idea of when the plotting started (exact) and when it is going to end (estimate).
![Screenshot 2021-05-10 at 14 18 17](https://user-images.githubusercontent.com/3693197/117658374-efbaaa80-b19a-11eb-9bd1-b91e5babd1ed.png)

**Comment by SackofFluor** [comment]
Just open a finished plotting log. There, at the very bottom it is written how long the whole plotting takes. As long as you don't go crazy on the amounts of parallel plots this number doesn't change significantly
